modules/shared-vpc/variables.tf

```diff
@@ -1,4 +1,9 @@
 variable "environment" {}
 variable "common_tags" {}
 variable "az_count" {}
-variable "database_availability_zones" {}
\ No newline at end of file
+variable "database_availability_zones" {}
+variable "cloudwatch_retention_period" {
```

Comment:
Could you append `_days` to the variable name so it clear what the "measurement" is to calling clients: `cloudwatch_retention_period_days`